openid-connect-4-credential-issuance-1_0.md

```diff
@@ -487,20 +521,17 @@ HTTP/1.1 200 OK
   Cache-Control: no-store
   Pragma: no-cache
 
-  {
-    "format": "jwt_vc"
-    "credential" : [
-      "LUpixVCWJk0eOt4CXQe1NXK....WZwmhmn9OQp6YxX0a2L" //Base64URL encoded VC
-    ]      
-  }
+{
+  "format": "jwt_vc"
+  "credential" : "LUpixVCWJk0eOt4CXQe1NXK....WZwmhmn9OQp6YxX0a2L"      
+}
```

Comment:
```yaml
CredentialResponse:
  title: Credential Response - synchronous flow
  description: Response that returns a Verifiable Credential in the requested format.
  type: object
  properties:
    format:
      description: JSON string denoting the credential format. It MUST be jwt_vc or ldp_vc.
      type: string
    credential:
      description: The issued credential(s).
      type: string
      format: Base64 encoded Verifiable Credential in the specified type.
    c_nonce:
      description: A random challenge that needs to be signed to prove the private key possession in case multiple VCs are issued.
      type: string
  required:
    - format
    - credential
 ```

Reply:
@alenhorvat could you create an openapi file for this draft once the request/responses have stabilized?